File: files/Average of N Numbers.c
#include<stdio.h>

int main()
{
    int i, c = 0, sum = 0, n;
    
    for (i = 0; n != -1; i++) {
        scanf ("%i", &n);
        sum = sum + n;
        c++;
    }
    int avg = sum / (c - 1);
    printf ("Avergae = %i", avg);
    
    return 0;
}

File: files/Average of N Numbers.java
import java.util.Scanner;
public class Main
{
    public static void main(String[] args) {
    Scanner uwu = new Scanner (System.in);
        
    int i, c = 0, n, sum = 0, avg;
    
        for (i = 0; n != -1; i++) {
            n = uwu.nextInt ();
            sum = sum + n;
            c++;
        }
        avg = sum / (c - 1);
        System.out.print ("Average = " + avg);
    }
}

